src/github/webhook.py

```diff
@@ -18,6 +18,8 @@ def _handle_pull_request_webhook(payload: dict) -> HttpResponse:
         pull_request = graphql_client.get_pull_request(pull_request_id)
         # a label change will trigger this webhook, so it may trigger automerge
         github_logic.maybe_automerge_pull_request(pull_request)
+        if payload["action"] == "closed":
```

Comment:
Put this check in so we don't delete branches on arbitrary PR updates.

Reply:
I suppose someone could create a branch with a new name, then re-open the same pull request and merge it which would result in a deletion but this seems like an unlikely situation so 👍